package.json

```diff
@@ -1,64 +1,80 @@
 {
-    "name": "sourcery",
-    "displayName": "Sourcery",
-    "description": "Refactor Python instantly with Sourcery",
-    "author": "Sourcery AI",
-    "repository": "https://github.com/sourcery-ai/sourcery-vscode",
-    "bugs": {
-        "url": "https://github.com/sourcery-ai/public/issues"
-    },
-    "license": "MIT",
-    "version": "0.8.0",
-    "sourceryVersion": "0.8.0",
-    "publisher": "sourcery",
-    "icon": "sourcery-icon.png",
-    "keywords": [
-        "Python",
-        "refactoring"
-    ],
-    "engines": {
-        "vscode": "^1.30.0"
-    },
-    "categories": [
-        "Other"
-    ],
-    "activationEvents": [
-        "onLanguage:python"
+  "name": "sourcery",
+  "displayName": "Sourcery",
+  "description": "Refactor Python instantly with Sourcery",
+  "author": "Sourcery AI",
+  "repository": "https://github.com/sourcery-ai/sourcery-vscode",
+  "bugs": {
+    "url": "https://github.com/sourcery-ai/public/issues"
+  },
+  "license": "MIT",
+  "version": "0.8.0",
+  "sourceryVersion": "0.8.0",
+  "publisher": "sourcery",
+  "icon": "sourcery-icon.png",
+  "keywords": [
+    "Python",
+    "refactoring"
+  ],
+  "engines": {
+    "vscode": "^1.30.0"
+  },
+  "categories": [
+    "Other"
+  ],
+  "activationEvents": [
+    "workspaceContains:*.py"
+  ],
+  "contributes": {
+    "commands": [
+      {
+        "command": "sourcery.refactor.workspace",
+        "title": "Scan with Sourcery",
+        "category": "Sourcery"
+      }
     ],
-    "contributes": {
-        "configuration": {
-            "title": "Sourcery Configuration",
-            "type": "object",
-            "properties": {
-                "sourcery.token": {
-                    "type": "string",
-                    "default": "",
-                    "description": "Sourcery token"
-                },
-                "sourcery.metricsEnabled": {
-                    "type": "boolean",
-                    "default": true,
-                    "description": "Show code metrics in the editor."
-                }
-            }
+    "menus": {
+      "explorer/context": [
+        {
+          "when": "explorerResourceIsFolder == true",
+          "command": "sourcery.refactor.workspace",
+          "group": "navigation"
```

Comment:
I don't know if this is the setting that chooses it but appearing right at the very top of the menu above `New File` seems to be too prominent to me?

Reply:
good catch - moved down to the next section